operator/Makefile

```diff
@@ -22,10 +22,16 @@ prep-release:
 	kustomize build ./crd > ./output/crd.yaml
 	ls -al ./output
 
-metacontroller/deploy:
+cert-manager/deploy:
```

Comment:
:question: Why was a `cert-manager` target added to the Makefile? It seems out of scope for deploying the Keip operator, since neither the metacontroller nor the keip webhook depend on it explicitly.

Reply:
The example route used TLS so cert-manager was needed to run it. Maybe that should just be entirely optional and I'll add back the readme steps to install cert-manager rather than doing it here.

Reply:
Yeah, I think that makes more sense, since technically the cert could from anywhere, so the use of cert-manager to create it is specific to this example.